### DISCUSSION TITLE
Clear filters button

### DISCUSSION THREAD

**Discussion by TheSoberJewel** [CAWSCIT/ca-meeting-finder-issue-tracker#25] [Ideas]
Please consider adding a button to allow the user to quickly clear/reset all search filters back to default. It is definitely something I have wished for while I have been playing with the app.

**Comment by enderv** [comment]
this would be an upstream request. we can ask tsml ui folks  if they are interested filed here https://github.com/code4recovery/tsml-ui/issues/381
